test/unittests/util/test_time.py

```diff
@@ -0,0 +1,76 @@
+from datetime import datetime
+from dateutil.tz import tzfile, tzlocal, gettz
+from unittest import TestCase, mock
+
+from mycroft.util.time import (default_timezone, now_local, now_utc, to_utc,
+                               to_local, to_system)
+
+test_config = {
+    'location': {
+        'timezone': {
+            'code': 'America/Chicago',
+            'name': 'Central Standard Time',
+            'dstOffset': 3600000,
+            'offset': -21600000
+        }
+    }
+}
+
+
+@mock.patch('mycroft.configuration.Configuration')
+class TestTimeFuncs(TestCase):
```

Comment:
future improvement: A more common strategy to testing time is to introduce the concept of a Clock as an interface, and provide deterministic clocks at test time. This prevents the necessity of mocking global statics, which can cause cascading failures through tests (and introduce bugs!) when misused.